Controls/UWPControls.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
Please, revert all changes made in the UWPControls.sln. A new configurations are added and VS version is changed - I believe this is made by VS automatically.